src/components/movie-card/movie-card.jsx

```diff
@@ -1,12 +1,44 @@
-// REACT
 import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+
 
 export class MovieCard extends React.Component {
   render() {
     const { movie, onClick } = this.props;
 
     return (
-      <div onClick={() => onClick(movie)} className="movie-card">{movie.Title}</div>
-    );
-  }
-}
\ No newline at end of file
+      <Card className="movie-card" style={{ width: '16rem' }}>
+      <Card.Img className="card-img" variant="top" src={movie.ImagePath} />
+      <Card.Body className="card-body">
+        <Card.Title className="card-title">{movie.Title}</Card.Title>
+        <Card.Text className="card-text">{movie.Description}</Card.Text>
+        <Button className="card-button" onClick={() => onClick(movie)} variant="link">Open</Button>
+      </Card.Body>
+    </Card>
+  );
+}
+}
+
+MovieCard.propTypes = {
+  movie: PropTypes.shape({
+    Title: PropTypes.string.isRequired,
+    Description: PropTypes.string.isRequired,
+
```

Comment:
Think about proper indentation here. It will make it much easier for you to read and understand the structure of the props, like this:
```
MovieCard.propTypes = {
	movie: PropTypes.shape({
		Title: PropTypes.string.isRequired,
		Description: PropTypes.string.isRequired,
		Genre: PropTypes.shape({
			Title: PropTypes.string.isRequired,
			Description: PropTypes.string.isRequired,
		}),
		Director: PropTypes.shape({
			Name: PropTypes.string.isRequired,
			Bio: PropTypes.string.isRequired,
			Birth: PropTypes.string.isRequired,
		}),
		ImagePath: PropTypes.string.isRequired,
	}).isRequired,
	onClick: PropTypes.func.isRequired
};

```

Reply:
![update movie-card](https://user-images.githubusercontent.com/71074389/103765876-2f6f5b00-501e-11eb-80bc-25fc8a460bd7.JPG)

Ok noted. As I use an extension "Prettier  - code formatter" on my VS Code, basically they do it organized for me automatically :) Otherwise i've change the structure manually.